### PR TITLE
postgresql: fix initialScript option

### PR DESCRIPTION
**Pull request by zagy**
The initialScript was never executed.

### CODE REVIEW

**Comment by domenkozar** [comment]
This just adds a text, where is the fix?

**Comment by zagy** [comment]
apparently I'm still too stupid to use git properly. Will retry tomorrow. Oh wait, later today.

**Comment by zagy** [comment]
I've commited the missing things. Sorry for that.

File: nixos/modules/services/databases/postgresql.nix [hunk 1]
```diff
@@ -198,6 +198,7 @@ in
                 if [ "$(id -u)" = 0 ]; then
                   chown -R postgres ${cfg.dataDir}
                   su -s ${pkgs.stdenv.shell} postgres -c 'initdb -U root'
+                  touch "${cfg.dataDir}/.first_startup"
```
Comment (domenkozar) [inline nixos/modules/services/databases/postgresql.nix]: this won't work for non-root operation

File: nixos/release.nix [hunk 2]
```diff
@@ -280,6 +280,7 @@ in rec {
   tests.openssh = callTest tests/openssh.nix {};
   tests.panamax = hydraJob (import tests/panamax.nix { system = "x86_64-linux"; });
   tests.peerflix = callTest tests/peerflix.nix {};
+  tests.postgresql.init = callTest tests/postgresql-init.nix {};
```
Comment (domenkozar) [inline nixos/release.nix]: Could you rename it to just `postgresql` test, we want everything related to postgresql to be in one test.

Comment (zagy) [inline nixos/release.nix]: Sure, will do.

File: nixos/modules/services/databases/postgresql.nix (hunk 1 above)
Comment (zagy) [inline nixos/modules/services/databases/postgresql.nix]: Ah, true. The .first_startup thingy should be moved out of the condition.

**Comment by zagy** [comment]
Better now?

**Comment by rickynils** [comment]
@zagy Can you rebase against master and squash your commits together? (http://nixos.org/nixpkgs/manual/#chap-submitting-changes)

**Comment by zagy** [comment]
@rickynils opened a new pull request, see #11083. (My knowledge around how to do things right with git is limited)

**Pull request closed without merging**